src/featbin/wav-reverberate.cc

```diff
@@ -0,0 +1,250 @@
+// featbin/extract-segments.cc
+
+// Copyright 2015  Tom Ko
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "base/kaldi-common.h"
+#include "util/common-utils.h"
+#include "feat/wave-reader.h"
+
+namespace kaldi {
+
+void FFTConvolution(Vector<BaseFloat> &x, Vector<BaseFloat> &h) {
+  int xdim = x.Dim();
+  int M = h.Dim();
+
+  KALDI_LOG << "M is " << M;
+
+  int N = 2;
```

Comment:
The  --verbose argument is added by default.
Dan


On Mon, Aug 17, 2015 at 1:20 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In src/featbin/wav-reverberate.cc
> <https://github.com/kaldi-asr/kaldi/pull/76#discussion_r37231927>:
>
> > +// See the Apache 2 License for the specific language governing permissions and
> > +// limitations under the License.
> > +
> > +#include "base/kaldi-common.h"
> > +#include "util/common-utils.h"
> > +#include "feat/wave-reader.h"
> > +
> > +namespace kaldi {
> > +
> > +void FFTConvolution(Vector<BaseFloat> &x, Vector<BaseFloat> &h) {
> > +  int xdim = x.Dim();
> > +  int M = h.Dim();
> > +
> > +  KALDI_LOG << "M is " << M;
> > +
> > +  int N = 2;
>
> Adding a verbosity argument to the binary and printing these statements
> when debugging level verbosity is required might be more desirable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/76/files#r37231927>.
>